pom.xml

```diff
@@ -4,31 +4,32 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.wildfly.swarm</groupId>
+    <groupId>io.thorntail</groupId>
     <artifactId>config-api-parent</artifactId>
-    <version>1.1.0.Final</version>
+    <version>1.4.0</version>
   </parent>
 
   <artifactId>keycloak-config-api-parent</artifactId>
-  <version>1.3.2-SNAPSHOT</version>
+  <version>1.4.0-SNAPSHOT</version>
 
   <packaging>pom</packaging>
 
   <properties>
     <version.keycloak>3.4.0.Final</version.keycloak>
-    <version.wildfly-config>1.3.0</version.wildfly-config>
+    <!-- We cannot use project.parent.version for submodules -->
+    <version.wildfly-config>1.4.0</version.wildfly-config>
   </properties>
 
   <scm>
-    <connection>scm:git:git@github.com:wildfly-swarm/keycloak-config-api.git</connection>
-    <developerConnection>scm:git:git@github.com:wildfly-swarm/keycloak-config-api.git</developerConnection>
-    <url>https://github.com/wildfly-swarm/keycloak-config-api</url>
+    <connection>scm:git:git@github.com:thorntail/keycloak-config-api.git</connection>
+    <developerConnection>scm:git:git@github.com:thorntail/keycloak-config-api.git</developerConnection>
+    <url>https://github.com/thorntail/keycloak-config-api</url>
     <tag>HEAD</tag>
   </scm>
 
   <modules>
-    <module>server</module>
     <module>client</module>
+    <module>server</module>
```

Comment:
Without this change the `keycloak-client-config-api` build always fails - for some reason the WildFly instance is shut down during exceution of `org.wildfly.swarm.config.generator.generator.Generator` and `wildfly-maven-plugin` fails to shut it down afterwards.